server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -268,7 +268,7 @@ cics_create: CREATE cics_create_opts;
 cics_create_opts:((ATOMSERVICE | BUNDLE | DB2CONN | DB2ENTRY | DB2TRAN | DOCTEMPLATE | DUMPCODE | ENQMODEL | FILE |
                    IPCONN | JOURNALMODEL | JVMSERVER | LIBRARY | LSRPOOL | MAPSET | MQCONN | MQMONITOR | PARTITIONSET |
                    PARTNER |  PIPELINE | PROCESSTYPE | PROFILE | PROGRAM | TCPIPSERVICE | TDQUEUE | TRANCLASS |
-                   TRANSACTION | TSMODEL | TYPETERM | URIMAP | WEBSERVICE | SESSIONS | TERMINAL | CONNECTION) cics_data_value |
+                   TRANSACTION | TSMODEL | TYPETERM | URIMAP | WEBSERVICE | SESSIONS | TERMINAL | CONNECTION) cics_data_value? |
```

Comment:
Only CONNECTION and TERMINAL seem to be impacted by this.

Reply:
Correct, these two commands are the ones that DISCARD or COMPLETE can be used and once one of these two options are in use , it is forbidden to use sub operand of belonging command. The opposite case is also valid.

Reply:
```suggestion
TRANSACTION | TSMODEL | TYPETERM | URIMAP | WEBSERVICE | SESSIONS) cics_data_value | (TERMINAL | CONNECTION) cics_data_value?
```